skrf/circuit.py

```diff
@@ -928,20 +937,110 @@ def X(self) -> np.ndarray:
         There is a numerical bottleneck in this function,
         when creating the block diagonal matrice [X] from the [X]_k matrices.
```

Comment:
not sure this comment is still True now. Can be removed if not.

Reply:
Fixed and detailed the performance bottleneck types of 'C' order and 'F' order.